{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -16,6 +16,8 @@ COPY ./src/requirements.txt requirements.txt
 {% if cookiecutter.use_alpine_linux == "y" -%}
 RUN grep psycopg2 requirements.txt && \
     apk add postgresql-dev gcc python3-dev musl-dev
+RUN grep git requirements.txt && \
```

Comment:
this grep needs to be more precise, `git://` or something

Reply:
@ppolewicz fixed in  https://github.com/reef-technologies/cookiecutter-rt-django/pull/22/commits/9d4c5e0f186ad476674c3d6e375c038cb8dc69f4